clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/datawriter/DataWriterTaskTracker.java

```diff
@@ -0,0 +1,106 @@
+package com.linkedin.venice.hadoop.task.datawriter;
+
+import com.linkedin.venice.hadoop.task.TaskTracker;
+
+
+/**
+ * An interface to report and retrieve metrics related to data writer tasks.
+ */
+public interface DataWriterTaskTracker extends TaskTracker {
```

Comment:
1. Should this be moved to some tracker package along with the 2 implementations?
2. 

> For MapReduce, there are two implementations of this interface:
ReporterBackedMapReduceDataWriterTaskTracker: which is used when the MR job is running
CounterBackedMapReduceDataWriterTaskTracker: which is used when the MR job is finished

Regarding this explanation, I am a bit confused. I think the ComputeJob has only one TaskTracker right? Or I am missing something...

Reply:
1. Currently, I have this packaging:
    a. com.linkedin.venice.hadoop.task.TaskTracker
    b. com.linkedin.venice.hadoop.task.datawriter.DataWriterTaskTracker
    c. com.linkedin.venice.hadoop.mapreduce.datawriter.task.CounterBackedMapReduceDataWriterTaskTracker
    d. com.linkedin.venice.hadoop.mapreduce.datawriter.task.ReporterBackedMapReduceDataWriterTaskTracker
    e.